instructure/canvas-ios
# Clear offline data on logout

refs: MBL-16706
affects: Student, Teacher, Parent
release note: none

test plan:
- Use simulator for manual testing.
- Delete the previously installed Student app.
- Log in to student app.
- Go to Developer menu, copy directory urls.
- Open a Finder window for the "App Directory" and for the "Shared Container Directory".
- Go to dashboard's "Manage Offline Content" screen.
- Select some files and sync them.
- Go to Finder and check the shared folder.
- There should be a Documents/domain-userId/Offline folder with the sqlite database files in it.
- Check the app's directory.
- There should be a Documents/domain-userId/Offline/Files folder and within folders with Canvas File IDs containing the actual files.
- In the app tap change user and switch back to the user you used.
- All the files should still be there in the Finder folders.
- Log out with the user.
- The domain-userId/Offline folders in both Finder windows should have been disappeared.
